core/driver/sources/src/main/resources/reference.conf

```diff
@@ -52,18 +52,27 @@ probe {
 
   // `resolvers` is for `org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers` - default values below
   resolvers {
-    intellij {
-      repositories = []
-    }
-    // default configuration for `plugins` is Plugins(None) - adding possible parameters in comments for reference
-    plugins {
-      // `repository.uri` is the URI of the plugin repository - use this config if you want to use a non-standard
-      // plugin repository. The standard one used by ide-probe is: https://plugins.jetbrains.com/plugin/download
-      # repository.uri = ""
-    }
-    jbr {
-      repositories = []
-    }
+    intellij.repositories = [
+      "https://download.jetbrains.com/idea/nightly/ideaIC-[revision].portable.zip",
+      "https://download.jetbrains.com/idea/ideaIC-[revision].portable.zip",
+      "https://www.jetbrains.com/intellij-repository/releases/[orgPath]/[module]/ideaIC/[revision]/ideaIC-[revision].zip",
+      "https://www.jetbrains.com/intellij-repository/snapshots/[orgPath]/[module]/ideaIC/[revision]/ideaIC-[revision].zip"
+    ]
+    // `plugins.repository.uri` is the URI of the plugin repository. Use this config if you want to use a non-standard
+    // plugin repository. The standard one is https://plugins.jetbrains.com/plugin/download
+    plugins.repository.uri = "https://plugins.jetbrains.com/plugin/download"
```

Comment:
Pls add some newlines to separate the keys visually (esp. needed for the places where HOCON isn't colored, like GitHub)